modules/distributor/distributor.go

```diff
@@ -32,9 +34,14 @@ import (
 const (
 	discardReasonLabel = "reason"
 
-	// RateLimited is one of the values for the reason to discard samples.
-	// Declared here to avoid duplication in ingester and distributor.
-	rateLimited = "rate_limited"
+	// reasonRateLimited indicates that the tenants spans/second exceeded their limits
+	reasonRateLimited = "rate_limited"
+	// reasonTraceTooLarge indicates that a single trace has too many spans
+	reasonTraceTooLarge = "trace_too_large"
+	// reasonTooManyTraces indicates that tempo is already tracking too many live traces in the ingesters for this user
+	reasonTooManyTraces = "too_many_traces"
```

Comment:
`too_many_traces` could also be interpreted as a rate limit.  Thoughts on an alternate string like `live_traces_exceeded` ?